src/api-service/__app__/onefuzzlib/azure/vmss.py

```diff
@@ -46,16 +46,10 @@ def list_vmss(name: UUID) -> Optional[List[str]]:
 def delete_vmss(name: UUID) -> bool:
     resource_group = get_base_resource_group()
     compute_client = get_compute_client()
-    try:
-        compute_client.virtual_machine_scale_sets.begin_delete(
-            resource_group, str(name)
-        )
-    except ResourceNotFoundError:
-        return True
-    except CloudError as err:
-        logging.error("cloud error deleting vmss: %s (%s)", name, err)
-
-    return False
+    response = compute_client.virtual_machine_scale_sets.begin_delete(
+        resource_group, str(name)
+    )
+    return bool(response.status() == "Succeeded")
```

Comment:
I suggest documenting the source of the `"Succeeded"` string, assuming there are no enums in play that mypy is leveraging.

Reply:
I can't find the documented value.  The API returns a poller of type None.  It's... not pleasant.

Reply:
added comments